package.json

```diff
@@ -6,16 +6,19 @@
   "scripts": {
     "clean": "rm -rf dist",
     "start": "node dist/main.js -p 8080 --log-level=debug",
-    "serve_local": "npm run build && nodemon dist/main.js -- --dev --log-level=silly --use-cache",
+    "serve_local": "npm run build && nodemon dist/main.js -- --dev --log-level=silly --port=8040",
```

Comment:
Here, I have removed caching and added the same port that webapp uses since that's usually what we're wanting.